pinot-tools/src/test/java/org/apache/pinot/tools/TestCreateSegmentCommand.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.tools;
+
+import com.google.common.base.Preconditions;
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import org.apache.pinot.spi.config.table.TableConfig;
+import org.apache.pinot.spi.config.table.TableType;
+import org.apache.pinot.spi.data.Schema;
+import org.apache.pinot.spi.data.readers.FileFormat;
+import org.apache.pinot.spi.utils.builder.TableConfigBuilder;
+import org.apache.pinot.tools.admin.command.CreateSegmentCommand;
+import org.testng.annotations.Test;
+
+
+public class TestCreateSegmentCommand {
+  private static final File JSON_INVALID_SAMPLE_DATA_FILE = new File(Preconditions
+      .checkNotNull(TestCreateSegmentCommand.class.getClassLoader().getResource("test_data/test_invalid_data.json"))
+      .getFile());
+
+  private static CreateSegmentCommand _createSegmentCommand = new CreateSegmentCommand();
+
+  @Test(
+    expectedExceptions = Exception.class,
+    expectedExceptionsMessageRegExp = "^.*test_invalid_data.json.*$"
+  )
+  public void testReadingInvalidJsonFile()
+      throws Exception {
+    final String fileDirectoryPath = JSON_INVALID_SAMPLE_DATA_FILE.getParent();
```

Comment:
(nit) We don't usually put `final` for local variables

Reply:
sure, btw is there a code style guide where I can check these things?

Reply:
We use the style config file located at `config/codestyle-intellij.xml` (`config/codestyle-eclipse.xml` for eclipse)
Will add some code style guide in the Pinot doc